components/uikit/Input/PasswordInput.tsx

```diff
@@ -0,0 +1,71 @@
+import React, { useState } from "react";
+import {
+  Box,
+  Input,
+  InputField,
+  InputSlot,
+  InputIcon,
+  FormControl,
+  AlertCircleIcon,
+  FormControlError,
+  FormControlErrorIcon,
+  FormControlErrorText,
+  FormControlHelper,
+  FormControlHelperText,
+} from "@gluestack-ui/themed";
+
+export type PasswordInputProps = {
+    error?: string;
+    help?: string;
+    isRequired?: boolean;
+    icon: React.ElementType;
+    value?: string;
+    onChange?: (value: string) => void;
+  };
+
+export const PasswordInput = ({error, help, isRequired, icon, value, onChange}:PasswordInputProps) => {
+    const [showPassword, setShowPassword] = useState(false)
+    const handleState = () => {
```

Comment:
Pour plus de clarté 
```suggestion
    const togglePasswordVisibility = () => {
```
NB: il te faudra changer le nom de la fonction au endroit où elle est utilisée.